specifications/Disruptor/Disruptor_SPMC.cfg

```diff
@@ -0,0 +1,16 @@
+SPECIFICATION
+    Spec
+
+CONSTANTS
+    Writers = { w }
+    Readers = { r1 r2 r3 }
```

Comment:
This should be ```{r1, r2, r3}```.

Reply:
Totally blunder - it was probably too late in the evening for me. :-P